test/f90_correct/test_many_backslash.f90

```diff
@@ -0,0 +1,45 @@
+! RUN: not %flang -c -fbackslash %s 2>&1 | FileCheck %s --check-prefix=CHECK-WITH-BSLASH-MSG1
+! RUN: not %flang -c -fbackslash %s 2>&1 | FileCheck %s --check-prefix=CHECK-MSG2
+! RUN: not %flang -c -fno-backslash %s 2>&1 | FileCheck %s --check-prefix=CHECK-MSG2
+
+program test1
+   print *, '\'
+End program test1
```

Comment:
NIT: Can the word `End` be lowercase like the rest of the program?

Reply:
fixed